### PR TITLE
Add created_at and updated_at to invoices and creditnotes endpoints

### PR DESCRIPTION
**Pull request by jdrieghe**
### Added
- `created_at` to `/invoices.list`
- `updated_at` to `/invoices.list`
- `created_at` to `/invoices.info`
- `updated_at` to `/invoices.info`
- `created_at` to `/creditNotes.list`
- `updated_at` to `/creditNotes.list`
- `created_at` to `/creditNotes.info`
- `updated_at` to `/creditNotes.info`

### CODE REVIEW

**Comment by luisaterrabrito** [comment]
What is the relation between the invoice_date, added_at and updated_at?

File: src/05-invoicing/invoices.apib [hunk 1]
```diff
@@ -156,7 +157,8 @@ Get details for a single invoice.
                     + paid_at: `2016-03-03T16:44:33+00:00` (string)
                     + payment (object)
                         + Include Money
-
+            + added_at: `2016-02-04T16:44:33+00:00` (string)
+            + updated_at: `2016-02-05T16:44:33+00:00` (string)
 ### invoices.draft [POST /invoices.draft]
```
Comment (jenssegers) [inline src/05-invoicing/invoices.apib]: Could use a newline here.

**Comment by jenssegers** [comment]
@luisabritotl; The invoice date is the date on the invoice without time, and can be in the future. Created at and updated at are CR(UD) related timestamps. The last 2 ones are read-only properties.

There is no direct relation between them.

**Review by luisaterrabrito — DISMISSED** [review]

**Review by mrtus — DISMISSED** [review]

**Review by jenssegers — APPROVED** [review]

**Review by steftrenson — DISMISSED** [review]

**Comment by jdrieghe** [comment]
This is live and can be merged imho

File: apiary.apib [hunk 2]
```diff
@@ -1564,7 +1564,8 @@ Get a list of invoices.
                             + rate: 0.21 (number)
                             + taxable (Money)
                             + tax (Money)
-
+                + added_at: `2016-02-04T16:44:33+00:00` (string)
```
Comment (jenssegers) [inline apiary.apib]: Would rename to `created_at`

**Review by jenssegers — DISMISSED** [review]
Could we rename `added_at` to `created_at` before we merge?

**Review by luisaterrabrito — APPROVED** [review]

**Review by mrtus — APPROVED** [review]

**Review by WouterSioen — APPROVED** [review]
Pretty sure linting is ok

**Comment by YorickH** [comment]
Manually checked by @WouterSioen

**Pull request merged**